packages/frontend-v2/hooks/usePlan.ts

```diff
@@ -1,35 +1,34 @@
-import useSWR, { SWRResponse } from "swr";
+import useSWR, {KeyedMutator, SWRResponse} from "swr";
```

Comment:
formatting xDDD

alright scrap telling u later, this is how to enable prettier format on save:

1. go to webstorm `preferences`
2. go to `plugins`. type in `prettier`. install it if it's not already installed (bundled)
3. search `prettier` in the preferences search bar (top left).
4. click on result under `languages and frameworks > javascript > prettier`. 
5. for `prettier package`, click dropdown, and select the version installed in the `graduatenu` folder (should be version 2.6.x roughly). 
6. make sure `run for files` looks something like this: `{**/*,*}.{js,ts,jsx,tsx,md}` (optional for u whether it includes md or not)
7. check the `on reformat code` action
8. check the `on save` action
9. click `apply` and then click `ok` in the bottom right.

for extra stuff on save:
1. from the above dialogue, click on `all actions on save`
2. check `optimize imports` and `run code cleanup`. `run prettier` should already be checked. i dont like including eslint because it somtimes takes more than a second to run (too long for me)
4. click `apply` and `save` again

Reply:
you're a life saver